rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,65 @@
+"""
+Relationships with Foreign Keys, esp. relationship() and FK values
+Why do we need to establish columns for players in game?
+difference with many to one and one to many -- if you back_populate either relationship type,
+    does it matter which you use?
+
+"""
+
+from sqlalchemy import Column, Integer, String
+from sqlalchemy import Enum
+from sqlalchemy import ForeignKey
+from sqlalchemy.ext.declarative import as_declarative
+from sqlalchemy.orm import relationship
+
+
+@as_declarative()
+class ModelBase(object):
+    def __tablename__(self):
+        return self.__class__.__name__.lower()
+
+
+class Game(ModelBase):
+    id = Column(Integer, primary_key=True)
+    player_1_id = Column(Integer, ForeignKey('user.id'))
+    player_2_id = Column(Integer, ForeignKey('user.id'))
+    round_id = Column(Integer, ForeignKey('round.id'))
+    rounds_choice = Column(Integer)
+    rounds_decrement = Column(Integer)
+    current_round = Column(Integer)
+    winner = Column(String, ForeignKey('user.id'))
+
+    round = relationship('Round', back_populates='game')
+    user = relationship('User', back_populates='game', uselist=False)
```

Comment:
If I'm getting too nitty gritty about the specifics about SQLAlchemy let me know, I'm unable to recognize if these details (esp. about relationships between models) are applicable to understanding how to write (better) Models and utilizing ORMs, like if this knowledge is universal.